src/TorchSharp/Utils/Histogram.cs

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static TorchSharp.torch;
+
+namespace TorchSharp.Utils
+{
+    // https://github.com/numpy/numpy/blob/v1.24.0/numpy/lib/histograms.py
```

Comment:
This is derived from Numpy and should therefore add an attribution notice, along with the copyright notice, to LICENSE in the root folder.

Reply:
Can you give me an example?

Reply:
@NiklasGustafsson 
For future TorchSharp users, many would use numpy-related codes, links, references,  this advice needs to be easily found

> This is derived from Numpy and should therefore add an attribution notice, along with the copyright notice, to LICENSE in the root folder

@ChengYen-Tang 

Are you using any Numpy for .NET libraries? If not, what advice would u give to new TorchSharp users who have numpy integration challenge with TorchSHarp?

Reply:
> Can you give me an example?

I was incorrect. It's in another file in the root folder: https://github.com/dotnet/TorchSharp/blob/main/THIRD-PARTY-NOTICES.TXT. The various sections of 3rd-party notices are separated by '==============' lines

Reply:
> @NiklasGustafsson For future TorchSharp users, many would use numpy-related codes, links, references, this advice needs to be easily found
> 
> > This is derived from Numpy and should therefore add an attribution notice, along with the copyright notice, to LICENSE in the root folder
> 
> @ChengYen-Tang
> 
> Are you using any Numpy for .NET libraries? If not, what advice would u give to new TorchSharp users who have numpy integration challenge with TorchSHarp?

@GeorgeS2019 
https://github.com/Quansight-Labs/numpy.net

Reply:
> > Can you give me an example?
> 
> I was incorrect. It's in another file in the root folder: https://github.com/dotnet/TorchSharp/blob/main/THIRD-PARTY-NOTICES.TXT. The various sections of 3rd-party notices are separated by '==============' lines.
> 
> Add another section, make a note that it's NumPy, and then copy the NumPy license text into the file.

Done.